draft-packages/avatar/docs/AvatarGroup.stories.tsx

```diff
@@ -31,84 +31,23 @@ DefaultStory.args = {
   avatars: avatarGroupData["users"],
 }
 
-export const DesignSheetDefault = () => {
+export const StickerSheetDefault = () => {
   const data = avatarGroupData["users"] as AvatarList
```

Comment:
I would probably do a similar thing to the other story where we include the data structure in the example here :)